### PR TITLE
Install types modules.

### PR DESCRIPTION
**Pull request by cqc-alec**

### CODE REVIEW

**Comment by Roland-djee** [comment]
Weird, I can't add my review.

**Comment by cqc-alec** [comment]
> Weird, I can't add my review.

Sorry, you were automatically added as a codeowner along with several others; I removed these and left @isobelhooper  as this affects her PR.

File: modules/pytket-honeywell/setup.py [hunk 1]
```diff
@@ -40,9 +40,11 @@
     install_requires=[
         "pytket ~= 0.11.0",
         "requests >= 2.2",
+        "types-requests",
         "websockets >= 7.0",
         "nest_asyncio >= 1.2",
         "pyjwt >= 1.7, < 2.0",
+        "types-jwt",
```
Comment (isobelhooper) [inline modules/pytket-honeywell/setup.py:47]: Looks good, although we'll need to remember to remove this should we use pyjwt 2.0+, according to https://pypi.org/project/types-jwt/.

**Review by isobelhooper — APPROVED** [review]
Changes look good!

File: modules/pytket-honeywell/setup.py (hunk 1 above)
Comment (cqc-alec) [reply modules/pytket-honeywell/setup.py:47]: Yes, good point.

**Pull request merged**